tests/nfs/nfs_operations.py

```diff
@@ -63,3 +63,84 @@ def cleanup_cluster(clients, nfs_mount, nfs_name, nfs_export):
 
     Ceph(clients[0]).nfs.export.delete(nfs_name, nfs_export)
     Ceph(clients[0]).nfs.cluster.delete(nfs_name)
+
+
+def create_files(client, mount_point, file_count):
+    """
+    Create files
+    Args:
+        clients (ceph): Client nodes
+        mount_point (str): nfs mount path
+        file_count (int): total file count
+    """
+    for i in range(1, file_count + 1):
+        try:
+            client.exec_command(
+                sudo=True,
+                cmd=f"dd if=/dev/urandom of={mount_point}/file{i} bs=1 count=1",
+            )
+        except Exception:
+            raise OperationFailedError(f"failed to create file file{i}")
+
+
+def perform_lookups(client, mount_point, num_files):
+    """
+    Perform lookups
+    Args:
+        clients (ceph): Client nodes
+        mount_point (str): nfs mount path
+        num_files (int): total file count
+    """
+    for _ in range(1, num_files):
+        try:
+            log.info(
+                client.exec_command(
+                    sudo=True,
+                    cmd=f"ls -laRt {mount_point}/",
+                )
+            )
+        except FileNotFoundError as e:
+            error_message = str(e)
+            if "No such file or directory" not in error_message:
+                raise OperationFailedError("failed to perform lookups")
+            log.warning(f"Ignoring error: {error_message}")
+        except Exception:
+            raise OperationFailedError("failed to perform lookups")
+
+
+def change_ownership(client, mount_point, file_count, user):
```

Comment:
I feel Mohit, since its not specific ti nfs, can we move change_ownership and change_permissions to utils? What are your thoughts?

Reply:
Thanks @pranavprakash20 
Done